### PR TITLE
Add information about adding validators without restarting.

### LINKED ISSUE #257: Doc loading validators without restarting
Dev PR: https://github.com/ConsenSys/teku/pull/3638

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Add information about adding validators without restarting.

## Fixed Issue(s)
fixes #257

### CODE REVIEW

File: docs/HowTo/Load-Validators-No-Restart.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    or ensure the URL supplied in [`--validators-external-signer-public-keys`](../Reference/CLI/CLI-Syntax.md#validators-external-signer-public-keys)
+    contains the new public keys.
+
+To load the validators in a running Teku instance, send a `SIGHUP` signal to the Teku process.
+
+```bash
+kill -HUP <PID>
+```
+
+Where `<PID>` is the process ID of the running Teku instance.
```
Comment (ajsutton) [inline docs/HowTo/Load-Validators-No-Restart.md:29]: We should probably add a warning that validators are never removed without a restart.

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -1800,3 +1808,4 @@ or clear your weak subjectivity settings.
 [slashing protection]: ../../Concepts/Slashing-Protection.md
 [weak subjectivity period]: ../../Concepts/Weak-Subjectivity.md
 [BeaconScan chain explorer]: https://beaconscan.com/ws_checkpoint
+[load new validators without restarting Teku]: ../../HowTo//Load-Validators-No-Restart.md
```
Comment (ajsutton) [inline docs/Reference/CLI/CLI-Syntax.md:1811]:
```suggestion
[load new validators without restarting Teku]: ../../HowTo/Load-Validators-No-Restart.md
```

File: docs/HowTo/Load-Validators-No-Restart.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Load-Validators-No-Restart.md:29]: Done.

File: docs/Reference/CLI/CLI-Syntax.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Syntax.md]: Fixed.

**Review by MadelineMurray — APPROVED** [review]

**Pull request merged**